app/code/Magento/CatalogUrlRewrite/Model/ProductScopeRewriteGenerator.php

```diff
@@ -21,6 +21,9 @@
 
 /**
  * Class ProductScopeRewriteGenerator
```

Comment:
Remove this line

Reply:
This was the original code but I've removed it anyway.